tests/framework/db/CommandTest.php

```diff
@@ -1525,4 +1525,21 @@ public function testBindValuesSupportsDeprecatedPDOCastingFormat()
         $db->createCommand()->setSql("SELECT :p1")->bindValues([':p1' => [2, \PDO::PARAM_STR]]);
         $this->assertTrue(true);
     }
+    
+    public function testBindValuesSupportsEnums()
+	{
+		if (version_compare(PHP_VERSION, '8.1.0') >= 0) {
```

Comment:
You can mark a test as "skipped", please see https://docs.phpunit.de/en/9.6/incomplete-and-skipped-tests.html#skipping-tests

```php
if (version_compare(PHP_VERSION, '8.1.0', '<')) {
   $this->markTestSkipped('Enums are not supported in PHP < 8.1');
}
```

Reply:
OK. Thank you.